prep.sh

```diff
@@ -0,0 +1,9 @@
+#! /bin/bash
+set xe
+
+curl https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 | bash
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
+curl -s https://raw.githubusercontent.com/rancher/k3d/main/install.sh | bash
+systemctl start docker
+systemctl enable docker.service
```

Comment:
Nevermind, I found it. Can we move away from `curl | bash` and vendor these scripts instead?

Reply:
created #14 to track this